src/OpenTelemetry.Instrumentation.SqlClient/CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## Unreleased
 
+* SqlClientInstrumentationOptions API changes: `SetDbStatementForStoredProcedure`,
+  `SetDbStatementForText` and `RecordException`
+  are available on all platforms. `SetDbStatement` has been removed.
+  On .NET Framework the following restrictions apply:
+  * `RecordException` has no effect
+  * `SetDbStatementForText` has no effect when using `System.Data.SqlClient`
```

Comment:
I think that's inconsistent with changes in SqlEventSourceListener.netfx.cs, or am I reading them wrong?

```csharp
(this.options.SetDbStatementForText || (eventData.EventSource == this.adoNetEventSource && this.options.SetDbStatementForStoredProcedure))
```

Will always capture if SetDbStatementForText = true, regardless of SDS vs MDS.

Reply:
Yes, you are right. With the change you suggest below we can actually make it do nothing on MDS as you can't capture text there and have SetDbStatementForStoredProcedure as the only setting that does something